sources/RuleRunner.cpp

```diff
@@ -85,7 +86,7 @@ status_t DeleteAction(const BMessage& action, entry_ref& ref);
 #undef B_TRANSLATION_CONTEXT
 #define B_TRANSLATION_CONTEXT "RuleRunner"
 
-#define PAIR(s)	{ s, B_TRANSLATE(s) }
+#define LOCALIZE(s)	{ s, B_TRANSLATE(s) }
```

Comment:
The `LOCALIZE` macro replaces e.g., `"Type"` with `{ "Type", B_TRANSLATE("Type") }` so that we don't need to code the string twice. The first, untranslated string initializes `english` of `NamePair` and the second, translated one initializes `locale` of `NamePair`.